modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala

```diff
@@ -197,13 +197,15 @@ object Databricks {
                   OPTIMIZE ${Fragment.const0(qualify(Manifest.Name))}
                   ZORDER BY base"""
               case Statement.CreateDbSchema =>
-                val schema = tgt.catalog.map(c => s"$c.${tgt.schema}").getOrElse(tgt.schema)
-                sql"""CREATE SCHEMA IF NOT EXISTS ${Fragment.const0(schema)}"""
+                sql"""CREATE SCHEMA IF NOT EXISTS ${Fragment.const0(qualifySchemaName)}"""
             }
 
-          private def qualify(tableName: String): String = tgt.catalog match {
-            case Some(catalog) => s"${catalog}.${tgt.schema}.$tableName"
-            case None => s"${tgt.schema}.$tableName"
+          private def qualify(tableName: String): String =
+            s"$qualifySchemaName.$tableName"
+
+          private def qualifySchemaName: String = tgt.catalog match {
+            case Some(c) => s"`$c`.${tgt.schema}"
```

Comment:
I've thought about surrounding schema with backticks as well however it wasn't possible to create schema with non-alphanumeric character in its name even when it is surrounded with backticks. Therefore, I've decided to pass it.